packages/rs-drive-abci/tests/strategy_tests/upgrade_fork_tests.rs

```diff
@@ -235,7 +229,7 @@ mod tests {
         } = run_chain_for_strategy(2000, hour_in_ms, strategy.clone(), config.clone(), 15);
         {
             let drive_cache = platform.drive.cache.borrow_mut();
-            let counter = drive_cache
+            let _counter = drive_cache
```

Comment:
Let's keep it?

Reply:
Yeah, that's exactly what it's doing - clippy won't be happy unless you named an unused variable with _, as by the style guide it signifies that that's a variable that's going to be used later

Reply:
Yeah, but I think this variable is here by mistake so I would keep this clippy warning until we fix it properly.